src/CSModule.sol

```diff
@@ -862,6 +887,43 @@ contract CSModule is
         });
     }
 
+    /// @inheritdoc ICSModule
+    function ejectBadPerformer(
+        uint256 nodeOperatorId,
+        uint256 keyIndex,
+        uint256[] calldata strikesData,
+        bytes32[] calldata proof
+    ) external {
+        _onlyExistingNodeOperator(nodeOperatorId);
+        NodeOperator storage no = _nodeOperators[nodeOperatorId];
+        if (keyIndex >= no.totalDepositedKeys) {
+            revert SigningKeysInvalidOffset();
+        }
+
+        uint256 pointer = _keyPointer(nodeOperatorId, keyIndex);
+        if (_isValidatorEjected[pointer]) revert AlreadySubmitted();
```

Comment:
Maybe?
```suggestion
        if (_isValidatorEjected[pointer]) revert AlreadyEjected();
```

Reply:
The current one is consistent with withdrawal submit, but we can change it